docs/docs/adrs/adr-020-cutomizable_slashing_and_jailing.md

```diff
@@ -0,0 +1,185 @@
+For example, validators double sining on consumer chains get slashed and are permanently jailed, 
+while validators not validating sufficient blocks are temporarily jailed.
+
+This means that ICS consumer chains get their economical security from the provider.
+However, this might come at a high cost.
+
+### The Cost of PoS
+
+One of the cost of validating on the consumer chains is operational -- validators need to run and monitor full nodes of every consumer chain they opt in for. 
+Although this cost varies from validator team to validator team (depending on how efficiently they can run their infrastructure), it doesn't depend on the total stake (or voting power) of the validators, so we can think of it as constant. 
+The other cost of validating comes from the risk of getting slashed or jailed.
+
+Most chains in Cosmos (including the Cosmos Hub) use delegated PoS -- users delegate their tokens to validators, which stake them in return for voting power. 
+Therefore, validators act as representatives chosen by their delegators to represent their interests. 
+However, delegators share the risk of their validators getting slashed or jailed:
+
+* When validators get slashed, a portion of their stake is being burned, including a portion of the tokens delegated by users.
+  As validators don't need to have their own stake, it is possible that delegators take all the risk of validators misbehaving.
+* When validators get jailed, they no longer receive block rewards (neither from the provider nor from the consumer chains). 
+  This also applies to their delegators. 
+  As a result, delegators might choose to restake their tokens with another validator.
+  The longer the validators are jailed, the more likely is that delegators will restake.
+  Thus, by getting jailed, validators risk damaging their reputation. 
+
+Misbehaviors don't need to be malicious, e.g., most cases of double signing infractions are due to misconfiguration. 
+This means that, by opting in on multiple consumer chains, validators and their delegators incur a higher risk.
+As a result, validators and their delegators want to be compensated for this additional risk, which makes the current design of ICS expensive. 
+
+This ADR addresses the high cost of ICS by allowing consumer chains to customize the slashing and jailing conditions. 
+Basically, every consumer chain can decide the punishment for every type of infraction. 
+This enables consumer chains to tradeoff economical security against cost.  
+
+## Decision
+
+To reduce the cost of ICS, consumer chains will be able to customize the slashing and jailing for every type of infraction. 
+As a result, consumer chains can decide on the amount of economic security they want and validators (and their delegators) can decide on the amount of additional risk they want to incur.  
+
+For every consumer chain, we introduce the following slashing and jailing parameters:
+```proto
+message InfractionParameters {
```

Comment:
Do we also want to parameterize (i.e., enable/disable) [fault resolutions](https://github.com/cosmos/interchain-security/pull/2051/files)?

Reply:
Fault resolutions are not implemented yet.